source/models/index.md

```diff
@@ -176,129 +139,13 @@ Frameworks (or apps built on those frameworks) don't need to write
-
-```js
-this.get('store').findRecord('person', 1); // => { id: 1, name: 'steve-buscemi' }
-```
-
-An ID is usually assigned to a record by the server when you save it for
-the first time, but you can also generate IDs client-side.
-
-## Adapter
-
-An **adapter** is an object that translates requests from Ember (such as
-"find the user with an ID of 1") into requests to a server.
-
-For example, if your application asks for a `Person` with an ID of
-`1`, how should Ember load it? Over HTTP or a WebSocket?  If
-it's HTTP, is the URL `/person/1` or `/resources/people/1`?
-
-The adapter is responsible for answering all of these questions.
-Whenever your app asks the store for a record that it doesn't have
-cached, it will ask the adapter for it. If you change a record and save
-it, the store will hand the record to the adapter to send the
-appropriate data to your server and confirm that the save was
-successful.
-
-Adapters let you completely change how your API is implemented without
-impacting your Ember application code.
-
-## Caching
-
-The store will automatically cache records for you. If a record had already
-been loaded, asking for it a second time will always return the same
-object instance. This minimizes the number of round-trips to the
-server, and allows your application to render its UI to the user as fast as
-possible.
-
-For example, the first time your application asks the store for a
-`person` record with an ID of `1`, it will fetch that information from
-your server.
-
-However, the next time your app asks for a `person` with ID `1`, the
-store will notice that it had already retrieved and cached that
-information from the server. Instead of sending another request for the
-same information, it will give your application the same record it had
-provided it the first time.  This feature—always returning the same
-record object, no matter how many times you look it up—is sometimes
-called an _identity map_.
-
-Using an identity map is important because it ensures that changes you
-make in one part of your UI are propagated to other parts of the UI. It
-also means that you don't have to manually keep records in sync—you can
-ask for a record by ID and not have to worry about whether other parts
-of your application have already asked for and loaded it.
-
-One downside to returning a cached record is you may find the state of
-the data has changed since it was first loaded into the store's
-identity map. In order to prevent this stale data from being a problem
-for long, Ember Data will automatically make a request in the
-background each time a cached record is returned from the store. When
-the new data comes in, the record is updated, and if there have been
-changes to the record since the initial render, the template is
-re-rendered with the new information.
-
-## Architecture Overview
+## How are server communications handled? Are they asynchronous?
 
 The first time your application asks the store for a record, the store
```

Comment:
@jenweber Is it possible to gloss over some details here? For example, throughout the document we haven't revealed the store's underlying abstraction (how it retrieves and store records) until now. We can also replace "persistence layer" with "back end" since that's the term you used throughout. Here's a first attempt at an update:

The first time you try to retrieve a record, the store checks if it has a local copy. If there's no local copy then a request will be made to the back end. This is an actual network call. If a local copy is found then it is returned immediately. Since the record could have changed on back end, between the first time it was requested and now, the store automatically refreshes it in the background. This is also a network call, although it can be prevented.